.dependabot/config.yml

```diff
@@ -4,6 +4,10 @@ update_configs:
     - package_manager: "dotnet:nuget"
       directory: "/exercises/accumulate"
       update_schedule: "live"
+      automerged_updates:
+        - match:
+            dependency_type: "all"
+            update_type: "semver:minor"
```

Comment:
Does this also wait for the CI to become green?

Reply:
Yes

https://dependabot.com/docs/config-file/#automerged_updates

> For all of the options below Dependabot will wait until all your status checks pass before merging.